iceoryx_hoofs/CMakeLists.txt

```diff
@@ -26,94 +26,106 @@ find_package(iceoryx_platform REQUIRED)
 include(IceoryxPackageHelper)
 include(IceoryxPlatformSettings)
 
-include(cmake/IceoryxHoofsDeployment.cmake)
-
 include("${CMAKE_CURRENT_LIST_DIR}/cmake/IceoryxPlatform.cmake")
 
+include(cmake/IceoryxHoofsDeployment.cmake)
+
 if(CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES Darwin)
     option(BUILD_SHARED_LIBS "Create shared libraries by default" ON)
 endif()
 
 set(PREFIX iceoryx/v${CMAKE_PROJECT_VERSION})
 
 #
-########## build iceoryx hoofs lib ##########
+# ######### build iceoryx hoofs lib ##########
```

Comment:
I guess the formatter interprets this as comment, maybe another separator symbol would be good

Reply:
The formatter forces a space between the comment mark and the being of the comment, I am disabling the format before this section as a solution,what do you think ?